fluidBuild.config.cjs

```diff
@@ -4,6 +4,9 @@
  */
 
 const tscDependsOn = ["^tsc", "^api", "build:genver", "ts2esm"];
+
+const checkFormattingTasks = ["prettier", "check:biome", "check:prettier"];
```

Comment:
Nit: since the associated script is called `check:format`, should this be called `checkFormatTasks` instead?

Reply:
And maybe add a comment about why we have `prettier` and `check:prettier`? Which if I'm getting this correctly, is to support build-tools which uses the latter, and everything else which uses the former?

Reply:
> why we have `prettier` and `check:prettier`?

It's temporary while the `prettier` tasks are replaced with `check:prettier` tasks (which may just be a rename of the existing task). A project should not have both tasks. The way task definitions in the config work is that they're considered deps _if_ they're defined in the project, otherwise they're ignored.

I'll add more comments to clarify.